openslides/assignments/static/templates/assignments/assignment-detail.html

```diff
@@ -177,53 +177,51 @@ <h3 translate>Election result</h3>
+              </button>
+            <!-- candidate name -->
+            <td>
+              <a ui-sref="users.user.detail({id: option.candidate.id})">{{ option.candidate.get_full_name() }}</a>
 
-          <!-- elected -->
-          <td class="minimum">
-            <button os-perms="assignments.can_manage"
-                ng-click="markElected(option.candidate_id, option.is_elected)" class="btn btn-default btn-xs">
-              <i ng-if="option.is_elected" class="fa fa-star" title="{{ 'is elected' | translate }}"></i>
-              <i ng-if="!option.is_elected" class="fa fa-star-o" title="{{ 'is not elected' | translate }}"></i>
-            </button>
-
-          <!-- candidate name -->
-          <td>
-            <a ui-sref="users.user.detail({id: option.candidate.id})">{{ option.candidate.get_full_name() }}</a>
-
-          <!-- votes -->
-          <td ng-if="poll.has_votes">
-            <div ng-init="votes = option.getVotes()">
-              <div ng-repeat="vote in votes">
-                <span ng-if="poll.yesnoabstain || poll.yesno">{{ vote.label }}:</span>
-                {{ vote.value }} {{ vote.percentStr }}
-                <div ng-if="vote.percentNumber">
-                  <uib-progressbar value="vote.percentNumber" type="success"></uib-progressbar>
+            <!-- votes -->
+            <td ng-if="poll.has_votes">
+              <div ng-init="votes = option.getVotes()">
+                <div ng-repeat="vote in votes">
+                  <span ng-if="poll.pollmethod == 'yna' || poll.pollmethod == 'yn'">{{ vote.label }}:</span>
+                  {{ vote.value }} {{ vote.percentStr }}
+                  <div ng-if="vote.percentNumber">
+                    <uib-progressbar value="vote.percentNumber" type="success"></uib-progressbar>
+                  </div>
                 </div>
               </div>
-            </div>
 
-        <!-- total votes (valid/invalid/casts) -->
-        <tr>
-          <td>
-          <td>
-            <translate>Valid votes</translate>
-          <td ng-if="poll.has_votes" ng-init="vote = poll.getVote(poll.votesvalid)">
-            {{ vote.value }} {{ vote.percent }}
-        <tr>
-          <td>
-          <td>
-            <translate>Invalid votes</translate>
-          <td ng-if="poll.has_votes" ng-init="vote = poll.getVote(poll.votesinvalid)">
-            {{ vote.value }}
-        <tr class="total bg-info">
-          <td>
-          <td>
-            <translate>Votes cast</translate>
-          <td ng-if="poll.has_votes" ng-init="vote = poll.getVote(poll.votescast)">
-            {{ vote.value }} {{ vote.percent }}
-      </table>
+          <!-- total votes (valid/invalid/casts) -->
+          <tr>
+            <td>
+            <td>
+              <translate>Valid ballots</translate>
+            <td ng-if="poll.has_votes" ng-init="vote = poll.getVote(poll.votesvalid)">
+              {{ vote.value }}
```

Comment:
The entered values of votesvalid, votesinvalid and votescast are empty in this template... I can't see why...